README.md

```diff
@@ -1,39 +1,60 @@
+
+## Structure
+- inside `src`:
+    - <u>`generate_svg.py` is the main script to generate the pdf. You can run this from command line or in a notebook.</u>
+    - `AllotaxChart.js` and `generate_svg_minimum.js` are the JavaScript files calculating the data and generating the plot in HTML before piping when `generate_svg.py` is run.
+    - `utils.py` contains helper functions to convert your data files between `csv`, `json`, and `js` formats if needed.
+    - `convert/`: test data
+    - `data/`: where you can place your data
+    - `output_charts/`: where the output html and pdf will be saved (see output examples here)
+- `allo_diagram.drawio` is a diagram of the allotaxonometer ecosystem.
+- `raw_requirements.txt` contains the raw requirements for the project which will eventually be used for packaging.
+- Once you set up your ecosystem, you will see `node_modules/`, which will contain the `npm` packages.
+
+
 
 ## Installation steps
 
-1. If environment install is needed (never used or installed `npm`, `nvm`, `node`):
-    - Here are the recommended [steps to install `nvm`](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating). `nvm` is a node version manager that streamlines installing the other 2.
-    - If you have `nvm`, you can do:
+1. If JavaScript tool installs are needed (never used or installed `npm`, `nvm`, `node`):
+    1. Here are the recommended [steps to install `nvm`](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating). `nvm` is a node version manager that streamlines installing the other
+
+    2. Once you have `nvm`, you can run this line to install the latest of both:
         ```shell
         nvm install --lts
         ```
-    to install the latest of both.
-    - Otherwise: [steps to install `node` and `npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
-1. One package depends on having Chrome (brower) or Chromium (browser driver) installed. If you have Chrome installed, you can skip this step. Otherwise, install Chrome or Chromium.
+    - Otherwise (not recommended): [steps to install individually install `node` and `npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+1. One package depends on having Chrome (brower) or Chromium (browser driver) installed. **If you have Chrome installed, you can skip this step**. Otherwise, install Chrome or Chromium.
 Here is the link for the [Chromium download](https://www.chromium.org/getting-involved/download-chromium/).
 
-1. Install the following:
+1. Clone this repo and install the requirements:
     ```shell
     git clone https://github.com/carterwward/py-allotax.git &&
     cd py-allotax &&
-    npm install d3 jsdom jstonge/allotaxonometer#py-allotax-experimental &&
+    npm install @ungap/structured-clone d3 jsdom jstonge/allotaxonometer#py-allotax-experimental &&
```

Comment:
I need to check, but if I merge `py-allotax-experimental` branch to the main, then publish it to npm. We could simply do `npm install allotaxonometer jsdom` (ignoring `d3` and `structured-clone`, as they are deps in `allotaxonometer`.

Reply:
Sounds good, happy to verify as a user on my end when we get to that point!